### PR TITLE
Bug/report post

### PR DESCRIPTION
**Pull request by Aboorva1**
Report submission notice is added

### CODE REVIEW

File: app/controllers/reports_controller.rb [hunk 1]
```diff
@@ -2,13 +2,13 @@ class ReportsController < ApplicationController
   def create
     report = Report.new (report_params)
     if report.save 
-      flash[:notice] = "Post reported"
-      respond_to do |format|
-        format.js
-        format.html {  redirect_to post_path(report.post_id, report.post) }
-      end
+      flash[:notice] = "Post is successfully reported"
```
Comment (iMagesh) [inline app/controllers/reports_controller.rb:5]: Please improve the message. Use something like, "Thanks for taking the time to report this post. Admins will be notified about this. "

Also send emails to Admins.

File: app/controllers/reports_controller.rb [hunk 2]
```diff
@@ -2,13 +2,13 @@ class ReportsController < ApplicationController
...
     else
-      render "form"
+      flash[:alert] = "Post is not reported"
```
Comment (iMagesh) [inline app/controllers/reports_controller.rb:7]: This error message can also be improved.

"Sorry something went wrong and this post was not reported. Can you please try again?"

**Review by iMagesh — CHANGES_REQUESTED** [review]
Please check comments and update

**Comment by iMagesh** [comment]
@Aboorva1 please make these changes so we can merge this PR

File: app/controllers/reports_controller.rb (hunk 1 above)
Comment (ksowndar) [reply app/controllers/reports_controller.rb]: The message is improved

File: app/controllers/reports_controller.rb (hunk 2 above)
Comment (ksowndar) [reply app/controllers/reports_controller.rb]: Here also the message is improved

File: db/seeds.rb [hunk 3]
```diff
@@ -41,3 +41,49 @@
...
+["Fraud","purpose of entertainment"].each do |reason|
+  ReportReason.create!(
+    report_category_id: 4,
+    reason: reason
+  )
+end
+
+["You","Someone else"].each do |reason|
+ReportReason.create!(
+  report_category_id: 5,
```
[the remark this answers is not in the document]
Comment (ksowndar) [reply db/seeds.rb:79]: For each category there are different reasons are in the reedit, so we need different report reasons for different report categories

**Comment by ksowndar** [comment]
> @Aboorva1 please make these changes so we can merge this PR
The changes are done @iMagesh

**Pull request merged**